numba_dpex/core/typing/dpnpdecl.py

```diff
@@ -179,12 +179,16 @@ def install_operations(cls):
         "ldexp",
         "spacing",
         "isnat",
+        "cbrt",
```

Comment:
Please split out the changes related to adding `cbrt` to unsupported list and removing `mod` from _aliases into a separate commit with meaningful commit message. This commit from description indicates only test cases were changes.